src/pointer-analysis/show_value_sets.h

```diff
@@ -12,10 +12,10 @@ Author: Daniel Kroening, kroening@kroening.com
 #ifndef CPROVER_POINTER_ANALYSIS_SHOW_VALUE_SETS_H
 #define CPROVER_POINTER_ANALYSIS_SHOW_VALUE_SETS_H
 
+#include <pointer-analysis/value_set_analysis.h>
```

Comment:
Why swap this to a full include when forward declaration was sufficient?

Reply:
`value_set_analysist` is a template specialisation now; `class value_set_analysist` won't work as the forward-decl will clash with the actual definition. There might be a way around this but I don't know it.